### PR TITLE
Malformed "format" Strings Cause Fatal

### PR DESCRIPTION
**Pull request by mitchute**
Pull request overview
---------------------
 - ```CoolingTower_VariableSpeed.idf``` was fataling due to a malformed format string. Four args were needed, but three were passed.
 - Fix ```vprint``` so malformed format strings don't cause fatal errors.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by Myoldmopar** [comment]
I hit this on a separate branch and pulled this change in and got the file running just fine.  I see this is still a draft PR, presumably because vprint could still have issues on other malformed formats without us really knowing it until it is too late.  I'd be happy to investigate if needed.

**Comment by mitchute** [comment]
I was planning to wait on finishing this until the state arg refactor drops to avoid all of the implications of having to pass that in. Or would you rather we just get this in first?

**Comment by Myoldmopar** [comment]
Let's just get this in as a hot fix and then deal with vprint later.  I think I'll mark this ready to review and get one more set of CI results to make sure nothing changed in develop to break it.  Then merge this in.

File: src/EnergyPlus/CondenserLoopTowers.cc [hunk 1]
```diff
@@ -4837,10 +4837,12 @@ namespace CondenserLoopTowers {
...
-                        state.dataCondenserLoopTowers->towers(this->VSTower).LGBuffer2 = format(" ...Valid maximum ratio = {:5.2F}. Occurrence info = {}, {} {}",
-                                                                 state.dataEnvrn->EnvironmentName,
-                                                                 state.dataEnvrn->CurMnDy,
-                                                                 General::CreateSysTimeIntervalString(state));
+                        state.dataCondenserLoopTowers->towers(this->VSTower).LGBuffer2 =
+                            format(" ...Valid maximum ratio = {:5.2F}. Occurrence info = {}, {} {}",
+                                   state.dataCondenserLoopTowers->towers(this->VSTower).MaxLiquidToGasRatio,
+                                   state.dataEnvrn->EnvironmentName,
+                                   state.dataEnvrn->CurMnDy,
+                                   General::CreateSysTimeIntervalString(state));
```
Comment (Myoldmopar) [inline src/EnergyPlus/CondenserLoopTowers.cc:4845]: Obvious fix here, and verified locally that it addresses the issue.

**Review by Myoldmopar — APPROVED** [review]

**Comment by Myoldmopar** [comment]
Uhh, file system test is totally unrelated to this error message tweak. This is good to go.  Thanks @mitchute!

**Pull request merged**